src/app/static/README.md

```diff
@@ -0,0 +1,27 @@
+## MINT Front End
```

Comment:
Great yes, we were missing this file before. Might be nice to add a link to this from the top level README, I think we have that in HINT.